docker/Dockerfile

```diff
@@ -3,11 +3,23 @@
 #
 
 FROM openebs/cstor-ubuntu:bionic-20200219
-RUN apt-get update; exit 0
-RUN apt-get -y install rsyslog
-RUN apt-get -y install curl tcpdump dnsutils net-tools iputils-ping gdb
-RUN apt-get -y install apt-utils libssl-dev libjson-c-dev libjemalloc-dev
-RUN apt-get -y install apt-file && apt-file update
+
+RUN apt-get update && \
+    apt-get install -y \
+    rsyslog \
+    curl \
+    tcpdump \
+    dnsutils \
+    net-tools \
+    iputils-ping \
+    gdb \
+    apt-utils \
+    libssl-dev \
+    libjson-c-dev \
+    libjemalloc-dev \
+    apt-file && apt-file update \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Can we move this command to new RUN since this is about cleaning the apt data.

Reply:
Same for arm64 docker file.

Reply:
If we move to the next layer then it will not impact image size as mentioned in best practices.

> In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Yes. My bad. It won't remove files from older layer unless you use `--squash` and  that is experimental feature, not supported by default.